alcs-frontend/src/app/services/application/decision/application-decision-v2/application-decision-condition/application-decision-condition.service.spec.ts

```diff
@@ -63,4 +63,63 @@ describe('ApplicationDecisionConditionService', () => {
     expect(mockHttpClient.patch).toHaveBeenCalledTimes(1);
     expect(mockToastService.showErrorToast).toHaveBeenCalledTimes(1);
   });
+
+  it('should make an http get and show a success toast when fetchPlanNumbers succeeds', async () => {
+    mockHttpClient.get.mockReturnValue(
+      of({
+        applicationFileNumber: '1',
+      }),
+    );
+
+    await service.fetchPlanNumbers('1');
+
+    expect(mockHttpClient.get).toHaveBeenCalledTimes(1);
+  });
+
+  it('should show a toast message if fetchPlanNumbers fails', async () => {
+    mockHttpClient.get.mockReturnValue(
+      throwError(() => {
+        new Error('');
```

Comment:
I'm not sure what a work around for this would be, but I think this could lend false confidence that an error toast is being shown for every type of failure, since we're only checking a specific case (there are a bunch of these).